### DISCUSSION TITLE
OpenID with Google Oauth

### DISCUSSION THREAD

**Discussion by Lordingard** [datalust/seq-tickets#2116] [Help]
Hello !
We need help about using OpenID to login with Google Wokspace accounts.
We arrive on the Google authentication page which tells us error 400 redirect_uri_mismatch. By analyzing we noticed that Seq was making http calls instead of making https calls, however we are well connected via https to the web interface. Do you have an idea ?
Thanks.

**Comment by nblumhardt** [comment]
Hi! You'll need to set:

```
SEQ_API_CANONICALURI=https://(your Seq address)
```

as an environment variable in the `datalust/seq` container (if you're using Docker), or:

```
seqcli config set -k api.canonicalUri -v https://(your Seq address)
```

from an admin PowerShell prompt if you're on Windows.

Hope this helps,
Nick
